openquake/commonlib/tests/logictree_test.py

```diff
@@ -476,6 +476,38 @@ def test_b_gr_relative_wrong_format(self):
         self.assertEqual(exc.message, 'expected single float value',
                          "wrong exception message: %s" % exc.message)
```

Comment:
For me yes, but I guess you also need a review from somebody in the scientific team.